src/main/java/uk/gov/hmcts/reform/sandl/snlrules/utils/DateTimeUtils.java

```diff
@@ -69,4 +70,47 @@ public static String humanizeDate(OffsetDateTime dateTime) {
         }
         return dateTime.toLocalDateTime().format(DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm"));
     }
+
+    public static boolean sameDay(OffsetDateTime one, OffsetDateTime two) {
+        OffsetDateTime startTruncated = one.truncatedTo(ChronoUnit.DAYS);
+        OffsetDateTime toCompareStartTruncated = two.truncatedTo(ChronoUnit.DAYS);
+
+        return !startTruncated.isAfter(toCompareStartTruncated) && !startTruncated.isBefore(toCompareStartTruncated);
+    }
+
+    public static boolean contains(OffsetDateTime biggerStart, OffsetDateTime biggerEnd,
```

Comment:
This method melts my mind a little bit, but I'm not sure if better naming for method o parameters can be applied.
```
(OffsetDateTime containerStart, OffsetDateTimecontainerEnd, OffsetDateTime toContainStart, OffsetDateTime toContainEnd)
```
but then again it doesn't solve a thing.